src/robusta/cli/integrations_cmd.py

```diff
@@ -58,13 +58,56 @@ def get_slack_key() -> (str, str):
 
 @app.command()
 def slack():
-    """generate slack api key"""
+    """Generate a Slack API key"""
     key, workspace = get_slack_key()
     log_title(
         f"Connected to Slack workspace {workspace}.\n"
         f"Your Slack key is:\n{key}\nAdd it to the slack sink configuration"
     )
 
 
+def get_ui_key() -> str:
+    """Generate a Robusta API key for the UI"""
+    while True:
+        email = typer.prompt(
+            "Enter a Gmail/Google Workspace address. This will be used to login"
+        )
+        email = email.strip()
+        account_name = typer.prompt("Choose your account name")
+
+        res = requests.post(
+            f"{backend_profile.robusta_cloud_api_host}/accounts/create",
+            json={
+                "account_name": account_name,
+                "email": email,
+            },
+        )
+        if res.status_code == 201:
+            robusta_api_key = res.json().get("token")
+            typer.secho(
+                "Successfully registered.\n",
+                fg="green",
+            )
+            return robusta_api_key
+
+        typer.secho(
+            f"Sorry, something didn't work out. The response was {res.content}\n"
+            f"If you need help, email support@robusta.dev",
+            fg="red",
+        )
+        try_again = typer.confirm("Would you like to try again?", default=True)
+        if not try_again:
+            return ""
+
+
+@app.command()
+def ui():
+    ui_key = get_ui_key()
+    if ui_key:
+        log_title(
+            f"Your UI key is:\n{ui_key}\nAdd it to the Robusta sink configuration"
```

Comment:
I would say 'add it as `token` to the Robusta sink configuration'

Reply:
I updated it so that it just prints the actual YAML you need. Is that OK?